lti_consumer/templates/html/lti_start_assessment.html

```diff
@@ -1,19 +1,14 @@
 {% load i18n %}
+
 <!DOCTYPE html>
 <html>
     <head>
-        <meta http-equiv="Content-Type" content="text/html; charset=UTF-8" >
+        <meta http-equiv="refresh" content="0; url='{{ context_url }}'" >
         <title>LTI</title>
     </head>
     <body>
         <p>
-            <b>{% trans "Sending you back to your exam." %}</b>
-        </p>
-        <script>
-            setTimeout(function () {window.close()}, 5000);
-        </script>
-        <p>
-            <a href="javascript:window.close();">
+            <a href="{{ context_url }}" target="_blank">
```

Comment:
Let me know if I should still keep this. My thought was if it didn't redirect right away. Currently, when testing, it just flashes this before going to the exam.

Reply:
I think there should be some text on the page saying "redirecting you back to your exam in 5 seconds" or something like that, so the user is aware of what's happening. It might also be useful to provide the user a direct link back to the exam (that renders in the same tab) just in case the redirect doesn't work. That UI pattern is pretty common!

Reply:
So I just updated this to basically look the same as it was before (so the copy is the same and there is the same 5 second delay).

Reply:
Ah you're right, good catch! Right now, line 6 is opening in the same tab and that link just opens in case that doesn't happen. But that was from the prior experience, so you're right I should update that! Thanks!